internal/install/install_test.go

```
@@ -7,36 +7,142 @@ import (
+			_, _, err := installer.Apply(context.Background(), test.options)
+			if err == nil || !strings.HasPrefix(err.Error(), "AGX-APPLY-BUNDLE-INPUT:") {
+				t.Fatalf("Apply() error = %v, want Bundle input error", err)
+			}
+		})
+	}
+}
+
+func TestApplyAcceptsInlineBundleAndPreservesReceiptBehavior(t *testing.T) {
+	archive := makeArchive(t, "source/README.md", []byte("inline\n"), tar.TypeReg)
+	server := httptest.NewTLSServer(http.HandlerFunc(func(writer http.ResponseWriter, _ *http.Request) {
+		writer.Write(archive)
```

golangci-lint (2.12.2)

[error] Error return value of `writer.Write` is not checked

(errcheck)